blocks/author-profile/author-profile.js

```diff
@@ -0,0 +1,71 @@
+import {
+  createOptimizedPicture,
+  getMetadata,
+} from '../../scripts/aem.js';
+import {
+  div, h2, p, a, span, img,
+} from '../../scripts/dom-builder.js';
+import ffetch from '../../scripts/ffetch.js';
+
+const AUTHOR_INDEX = '/authors-index.json';
+const fallbackPath = '/author/name';
+const fallbackName = 'Name';
+const fallbackDescription = 'Author';
+const fallbackLinkText = 'See more by this author';
+const fallbackLinkAriaLabel = 'Read more';
```

Comment:
Also these should not be extra constants.

Reply:
Inlined.